packages/yourloops/test/integration/pages/home-page/hcp-home-page.spec.tsx

```diff
@@ -144,6 +181,7 @@ describe('HCP home page', () => {
   })
 
   it('should allow to remove a patient who is in multiple teams', async () => {
```

Comment:
I don't think that this test is relevant anymore (as we cannot choose the team from the dropdown anymore)